pom.xml

```diff
@@ -136,6 +136,16 @@
                 <artifactId>bigdata-core</artifactId>
                 <version>${blazegraph.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-core</artifactId>
+                <version>${jackson.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-annotations</artifactId>
```

Comment:
I cannot see where this dependency is used. Is it really needed?

This dependency should be declared before jackson-core.

Reply:
The jackson-annotations and jackson-core are pulled by jackson-databind (hence the order, but we can change it).

In case we don't impose here the version used, blazegraph pulls its own version of those dependencies, which is inconsistent with our version of jackson-databind : it throws unknown methods exceptions at runtime in cgmes unit tests.

As it could happen with dependencies other than blazegraph, it's better to impose our own version of those 2 dependencies a the parent pom level.

Reply:
It's now correctly sorted!